save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -699,6 +712,44 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
         }
     }
 
+    private fun updateGit() {
+        scope.launch {
+            val headers = Headers().also {
+                it.set("Accept", "application/json")
+                it.set("Content-Type", "application/json")
+            }
+
+            val git = GitDto(
+                fieldsGitMap[InputTypes.GIT_URL]?.trim() ?: gitDto!!.url,
```

Comment:
Please rename “fieldsGitMap” is a bad name. What did you mean? gitCredentials? gitInfo? fieldsWithGitInfo?

Reply:
Renamed